payment_methods/Paypal_Express/forms/SettingsForm.php

```diff
@@ -17,53 +14,22 @@
  * @author         Mike Nelson
  * @since          4.9.59.p
  */
-class SettingsForm extends EE_Payment_Method_Form
+class SettingsForm extends PayPalSettingsForm
 {
-
-    /**
-     * @var string of HTML being the help tab link
-     */
-    protected $helpTabLink;
-
     /**
      * SettingsForm constructor.
      *
      * @param array $options_array
-     * @param string     $help_tab_link
+     * @param string $help_tab_link
+     * @throws \EventEspresso\core\exceptions\InvalidDataTypeException
+     * @throws \EventEspresso\core\exceptions\InvalidInterfaceException
+     * @throws \InvalidArgumentException
```

Comment:
these classes should all be imported, so that you can see everything the class utilizes via the `use` statements at the top of the file

Reply:
oups, missed those. Thanks.